src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -723,4 +767,365 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 		}
 
 	}
+
+	//my code here:
+	private createNewTable(): void {
+
+		let t: Table<any>;
+		if (this.placeHolderDataSets) {
+			let dataSet = this.placeHolderDataSets[0];
+			let options = {
+				enableCellNavigation: true,
+				enableColumnReorder: false,
+				renderRowWithRange: true,
+				showHeader: true,
+				rowHeight: this._rowHeight,
+				defaultColumnWidth: 120,
+				editable: false,
+				autoEdit: false,
+				enableAddRow: false, // TODO change when we support enableAddRow
```

Comment:
do we know what this `TODO` is about?  If not, perhaps we should remove it?

Reply:
I have no idea, it was there to begin with.